contracts/Migrations.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity >=0.4.21 <0.6.0;
```

Comment:
do we need to lock this down to a 0.5 version like safeexchange? ` pragma solidity ^0.5.2; `